src/database/models/event_item.rs

```diff
@@ -0,0 +1,285 @@
+use super::{
+    dynamic::{DynamicId, IdType},
+    generate_event_id, DatabaseError, EventId, OrganizationId, ProjectId, UserId,
+};
+use chrono::{DateTime, Utc};
+use itertools::Itertools;
+use sqlx::postgres::{PgHasArrayType, PgTypeInfo};
+use std::convert::{TryFrom, TryInto};
+
+#[derive(sqlx::Type, Clone, Copy)]
+#[sqlx(type_name = "text")]
+#[sqlx(rename_all = "snake_case")]
+pub enum EventType {
+    ProjectCreated,
```

Comment:
version created/ project updated event is missing

Reply:
I wasn't under the impression we wanted those implemented in this first pass and assumed we would follow up this PR with ones implementing those.

Would you like me to implement those in this PR?

Reply:
Yeah I think it would be worth implementing